playbooks/provision.yml

```diff
@@ -69,3 +69,10 @@
 
     - role: webserver # Build the unicorn webserver.
       tags: webserver
+
+    - role: Datadog.datadog
+      become: yes
+      vars:
+        datadog_api_key: "{{ datadog_key }}"
```

Comment:
In the PR descriptio you talk about `datadog_api_key` variable, which one is correct?

In any case would be nice to add it to https://github.com/openfoodfoundation/ofn-install/blob/master/inventory/host_vars/_example.com/secrets.example.yml

Reply:
Update the description, sorry

Reply:
Could you please update the secrets example? thanks!

Reply:
Done!